packages/swapper/src/swappers/zrx/getZrxSupportedAssets/getZrxSupportedAssets.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('getZrxSupportedAssets', () => {
+  it('is true', () => {
+    expect('yes').toEqual('yes')
```

Comment:
Not sure about this, maybe a leftover ? :) even if its very basic we can unit test `getSupportedAssets` and `isSupportedAssets` methods